### DISCUSSION TITLE
Integrating StyleX compiler into Storybook NextJS preset

### DISCUSSION THREAD

**Discussion by shawngustaw** [storybookjs/storybook#25390] [Help]
### Summary

Was wondering if anyone has got Storybook/Next/StyleX all working together correctly, and if not, if anyone was willing to try to help figure out a way to integrate their Webpack/Babel plugins into Storybook?

I have a pretty barebones next app with stylex + storybook here: https://github.com/shawngustaw/nextjs-app-dir-stylex

The next plugin can be found here: https://github.com/facebook/stylex/blob/main/packages/nextjs-plugin/src/index.js 

Been a while since I've touched webpack/babel stuff so any help would be greatly appreciate if anyone gets this working!

**Comment by ShaunEvening** [comment]
hey Shawn!

I can probably help out with this this week 😁 

I just need to catch up on my notifications from the holidays first.
